ansible/group_vars/all.yml

```diff
@@ -5,6 +5,8 @@
 rhevm_username: admin@internal
 rhevm_password: smartvm
 export_datastore: export
+rhevm_hostname: performance-testing-domain.com
```

Comment:
Lets adjust this to rhevm_host to "match" the naming convention we have for carbon and grafana below.  Also lets put this above username to match style wise as well.